tests/Sphere3d.elm

```diff
@@ -202,6 +202,70 @@ properties =
         )
 
 
+scaleAbout : Test
+scaleAbout =
+    fuzz3 point3d
+        scalar
+        sphere3d
+        "A sphere gets scaled correctly"
+        (\point scale sphere ->
+            sphere
+                |> Sphere3d.scaleAbout point scale
+                |> Expect.equal
+                    (Sphere3d.with
+                        { radius = abs (scale * Sphere3d.radius sphere)
+                        , centerPoint = Point3d.scaleAbout point scale (Sphere3d.centerPoint sphere)
+                        }
+                    )
+        )
+
+
+rotateAround : Test
+rotateAround =
+    fuzz3 axis3d
+        scalar
+        sphere3d
+        "A sphere gets rotated correctly"
+        (\axis angle sphere ->
+            sphere
+                |> Sphere3d.rotateAround axis angle
+                |> Expect.all
```

Comment:
Do you prefer `Expect.all` (this test) or `Expect.equal (Sphere3d.with)` (scaleAbout)?

Reply:
I guess between the two I slightly prefer the `Expect.equal` version since it seems a bit simpler/more readable, although I think I'd use `Expect.sphere3d` instead to allow a tiny bit of tolerance - small differences in implementation could result in rounding differences between the test and implementation. The `Expect.all` version seems more flexible, though, for checking various properties of the result. So maybe `Expect.sphere3d` etc. where that's all you need, and `Expect.all` if you're doing something more complex?

In general, I think it's generally considered best practice to try *not* to have the test mirror the implementation - for example, the `Point3d.rotateAround` tests check for stuff like "the original point and rotated point should be the same distance from and along the rotation axis", not "the coordinates of the resulting point are equal to this rotation matrix times the coordinates of the original point", and you could add more elaborate stuff like "if you construct the direction from the axis to the original point and from the axis to the rotated point, the angle between the two directions should be equal to the absolute value of the given rotation angle" (at least for rotation angles less than 180 degrees). In this case I don't see a way to do anything like that, so I think what you've done is fine, but it's something to consider for other tests in the future.